sparse_strips/vello_cpu/src/dispatch/multi_threaded.rs

```diff
@@ -488,6 +498,28 @@ impl Dispatcher for MultiThreadedDispatcher {
             }
         }
     }
+
+    fn generate_wide_cmd(
+        &mut self,
+        strip_buf: &[Strip],
+        fill_rule: Fill,
+        paint: Paint,
+        thread_idx: u8,
```

Comment:
Can this ever be not 0? If it will always be 0 atm, should we disallow setting that value for now and just hardcode 0 below when registering the task?